### PR TITLE
Plan Your Attack

### PR DESCRIPTION
**Pull request by wylie**
Cleaning up the CodeGuide. It's been on the back burner for a little while and things have gotten a little out-of-date.
- Cleaning up the section numbers
- Removing some that shouldn't, in the long run, be in Edna
- Reorganizing the list

/cc @gjjones

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -167,7 +167,30 @@ module.exports = function(grunt) {
 					watchTask: true
```
Comment (wylie) [inline Gruntfile.js:167]: Why does this file continue to show up in my PRs...?

File: less/_input-group.less [hunk 2]
```diff
@@ -35,22 +35,22 @@
 //     <span class="input-group-btn">
 //         <button class="btn disabled">Click me!</button>
 //     </span>
-//     <input type="text" placeholder="placeholder text" disabled />
+//     <input class="input-txt" type="text" placeholder="placeholder text" disabled />
```
Comment (wylie) [inline less/_input-group.less:38]: this class was missing

File: Gruntfile.js (hunk 1 above)
Comment (gjjones) [inline Gruntfile.js:167]: It looks like you are changing something, then undoing the change, then changing it again, not you're undoing it again

**Comment by gjjones** [comment]
:+1:

**Pull request merged**